src/current/cockroachcloud/sql-shell.md

```diff
@@ -0,0 +1,100 @@
+---
+title: SQL Shell
+summary: Use Cloud Console SQL Shell to run statements.
+toc: true
+cloud: true
+docs_area: deploy
+---
+
+{{site.data.alerts.callout_info}}
+{% include_cached feature-phases/limited-access.md %}
+{{site.data.alerts.end}}
+
+The **SQL Shell** page on the {{ site.data.products.db }} Console enables you to run queries on your cluster directly from your browser.
+
+To use this feature, select a cluster from the [**Clusters** page](cluster-management.html#view-clusters-page), and navigate to the cluster's **SQL Shell** page.
+
+## Limitations
+
+- All statements in the SQL Shell are executed within a transaction, so statements like [`SET CLUSTER SETTING`](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/set-cluster-setting) are not supported.
+- The SQL Shell does not yet support sessions.
+- The SQL Shell is available only to users of enrolled organizations with the [Cluster Administrator or Cluster Operator role]({% link cockroachcloud/managing-access.md %}).
+
+## Overview
+
+Above the SQL Shell input field, you will see the active user and cluster details in the format `{user name} @ {cluster-name}:{database}`. Note that the user is the **Team member** currently logged into the {{ site.data.products.cloud }} Console, not a SQL user, and you must have the Cluster Administrator or Cluster Operator [role]({% link cockroachcloud/authorization.md %}#organization-user-roles) to use this SQL Shell. Other users can still access CockroachDB's [command line SQL shell](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/cockroach-sql).
```

Comment:
We can specify which SQL user is being used (is it `root`?)

Reply:
Yes